docs/source/roadmap.rst

```diff
@@ -16,28 +16,23 @@ contributions. Here's our plans for the next two releases.
     the feature. Pull Requests are open to anyone.
 
 
-3.2
----
-
-* **RHEL/CentOS:** Support RHEL 8.x (assuming it has been released!)
-* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
-* **ChatOps:** Support RBAC.
-* **Orquesta:** Support task retry in workflow definition.
-* **Orquesta:** Support rerun of workflow execution from task(s).
-
 3.3
 ---
 
-* **Ubuntu:** Stop producing Ubuntu 14.04 packages.
+* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
 * **Mistral deprecation:** Orquesta replaces Mistral as the workflow engine.
 
+
 Monitor the `master branch <https://github.com/StackStorm/st2/>`_ to see how we're progressing.
 
 Backlog
 -------
 
-Here's some more things on our list that we haven't scheduled yet:
+Here's some more nice things on our list that we haven't scheduled yet:
 
+* **Python:** Drop Python 2 support, rely on Python 3 ony.
```

Comment:
@blag RE `py2` vs `py3`.
There were some scattered messages, but I don't remember these kind of agreed plans for any release.
Sounds like it's a good topic for the next TSC meeting. Please raise it. I like the goal too :+1: 

Removing py 2.7 won't be that easy as it'll affect `U16` and `EL7` and packaging we currently use.
Probably worth a more in-depth technical @StackStorm/tsc conversation what's technically involved there and commitments around this item.

Reply:
Gotcha, that makes sense. I'll add it to the docket for the next TSC meeting.